pyseir/ensembles/ensemble_runner.py

```diff
@@ -58,17 +73,102 @@ def __init__(self, fips, n_years=2, n_samples=250,
         self.n_years = n_years
         self.t0 = fit_results.load_t0(fips)
         self.date_generated = datetime.datetime.utcnow().isoformat()
-        self.suppression_policy = suppression_policy
+        self.run_mode = RunMode(run_mode)
+        self.state_abbr = us.states.lookup(self.county_metadata['state']).abbr
 
+        self.suppression_policy = suppression_policy
         self.summary = copy.deepcopy(self.__dict__)
         self.summary.pop('t_list')
         self.generate_report = generate_report
 
+        self.suppression_policies = None
+        self.override_params = None
+        self.init_run_mode()
+
         self.all_outputs = {}
         self.output_file_report = os.path.join(OUTPUT_DIR, self.county_metadata['state'], 'reports',
-            f"{self.county_metadata['state']}__{self.county_metadata['county']}__{self.fips}__ensemble_projections.pdf")
+            f"{self.county_metadata['state']}__{self.county_metadata['county']}__{self.fips}__{self.run_mode.value}__ensemble_projections.pdf")
         self.output_file_data = os.path.join( OUTPUT_DIR, self.county_metadata['state'], 'data',
-            f"{self.county_metadata['state']}__{self.county_metadata['county']}__{self.fips}__ensemble_projections.json")
+            f"{self.county_metadata['state']}__{self.county_metadata['county']}__{self.fips}__{self.run_mode.value}__ensemble_projections.json")
+
+    def init_run_mode(self):
+        """
+        Based on the run mode, generate suppression policies and ensemble
+        parameters.  This enables different model combinations and project
+        phases.
+        """
+        self.suppression_policies = dict()
+
+        min_date_data = datetime.datetime.today() - datetime.timedelta(days=30)
```

Comment:
Does this define the earliest date extract case data or predictions?

Reply:
Outdated. No longer needed, but it is the earliest data extracted.